package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "frigg-monorepo",
-  "version": "1.0.4",
+  "version": "1.0.0",
```

Comment:
Does down-bumping this matter at all? I think the answer is "no", because this isn't actually packaging/releasing, right? This could stay unversioned?

Reply:
yes I don't think it should matter, though I'm not sure... but it seems like a good idea try to release on 1.0.0